### PR TITLE
Add #rcount command, the number of bans from #roulette

### PR DESCRIPTION
**Pull request by dasdgdafg**
The initial value is computed by checking the mod-log.  After that, the counter is incremented each time someone is banned by #roulette

hxixzj

### CODE REVIEW

File: client_cpp/src/posts/models.hh [hunk 1]
```diff
@@ -101,6 +102,7 @@ public:
     std::vector<uint16_t> dice; // Result of dice throw
     std::string eight_ball; // Result of #8ball command
     unsigned int roulette[2]; // Result of #roulette command
+    unsigned long rcount; // Deaths from #roulette
```
Comment (bakape) [inline client_cpp/src/posts/models.hh]: No reason to add this. Just reuse the same field pyu does.

File: go/src/meguca/common/commands.go [hunk 2]
```diff
@@ -54,6 +58,7 @@ type Command struct {
 	Eightball string
 	Dice      []uint16
 	Roulette  [2]uint8
+	Rcount    uint64
```
Comment (bakape) [inline go/src/meguca/common/commands.go]: Same here.

File: go/src/meguca/db/sql/posts/increment_rcount.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+update main
```
Comment (bakape) [inline go/src/meguca/db/sql/posts/increment_rcount.sql]: There will be no `id = 'rcount'` row in the DB, if the DB was generated freshly and not from a previous version upgrade. Probably need to init the row to 0 in `db/init.go:initDB()`. 

Now that I think about it, the same will happen with current roulette status, so mind initing that to 0 too?

Comment (dasdgdafg) [reply go/src/meguca/db/sql/posts/increment_rcount.sql]: roulette gets initialized in init.sql.  I'll add rcout there too.

Comment (bakape) [reply go/src/meguca/db/sql/posts/increment_rcount.sql]: Oh, did not notice. Good then.

**Pull request merged**